lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -283,6 +288,20 @@ states:
       delete:
         targetState: DELETE_HANDOVER_PAGE
 
+  COULD_NOT_UPDATE_DETAILS_PAGE_RFC:
+    response:
+      type: page
+      pageId: update-details-failed
+      context: repeatFraudCheck
+    events:
+      delete:
+        targetState: DELETE_HANDOVER_PAGE
+      back:
```

Comment:
I'm not sure about overloading the 'back' event here - IMO we should try to reserve that for a 'back' button only